### DISCUSSION TITLE
[Enhancement]: Changelog

### DISCUSSION THREAD

**Discussion by ofcdune** [ciderapp/Cider#659] [Feature Request]
### The feature you would like

Hey guys, I really enjoy your app and I really enjoy that you are pushing so many updates on a regular basis. Because I am a tech- and Apple Music enthusiast, I would like to have a changelog to see on the spot what you have updated.

Thank you so much and best wishes,
dune-official

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by coredev-uk** [comment]
A changelog would be ideal in the future, but as the application is still in development and lots of areas of the app are changing, a changelog would take up a lot of our time. If you want to view features that users requested that have been implemented, take a look at our project board.

**Comment by quacksire** [comment]
Here's the current auto generated changelog (github does this)

## What's Changed
* Fix issue with empty "Friends listening to" box by @N0chteil in https://github.com/ciderapp/Cider/pull/97
* Fix Acrylic v2 by @yazninja in https://github.com/ciderapp/Cider/pull/110
* get rid of acrylic by @saboooor in https://github.com/ciderapp/Cider/pull/120
* Added a confirm button so you dont accidently delete a playlist by @Swiftzerr in https://github.com/ciderapp/Cider/pull/119
* Get rid of acrylic in settings by @Swiftzerr in https://github.com/ciderapp/Cider/pull/121
* merge by @vapormusic in https://github.com/ciderapp/Cider/pull/124
* Fixed #114 by @Swiftzerr in https://github.com/ciderapp/Cider/pull/123
* merge by @vapormusic in https://github.com/ciderapp/Cider/pull/126
* merge from main by @vapormusic in https://github.com/ciderapp/Cider/pull/133
* Show lyrics button only if there are lyrics by @yazninja in https://github.com/ciderapp/Cider/pull/125
* change of the ugly "AUTO" button to the ∞ by @gms10ur in https://github.com/ciderapp/Cider/pull/135
* merge from main by @vapormusic in https://github.com/ciderapp/Cider/pull/136
* m by @vapormusic in https://github.com/ciderapp/Cider/pull/137
* Added thumbar buttons by @gms10ur in https://github.com/ciderapp/Cider/pull/138
* move thumb button by @vapormusic in https://github.com/ciderapp/Cider/pull/140
* Change hyphens into em dashes for separators by @SoNothingMC in https://github.com/ciderapp/Cider/pull/143
* Add explicit icon next to song name by @JaxonTekk in https://github.com/ciderapp/Cider/pull/144
* Minor UX Improvements: Context Menu Position and Album Hitbox by @cyraki in https://github.com/ciderapp/Cider/pull/149
* misc improvements by @JaxonTekk in https://github.com/ciderapp/Cider/pull/145
* Add main branch renderer edits. by @cryptofyre in https://github.com/ciderapp/Cider/pull/151
* styled volume slider for fullscreen display by @JaxonTekk in https://github.com/ciderapp/Cider/pull/152
* Context Menu - Library Artist Item by @cyraki in https://github.com/ciderapp/Cider/pull/157
* Saving of hide/show user info & minor ui tweaks by @gms10ur in https://github.com/ciderapp/Cider/pull/158
* Song duration alignment by @maikirakiwi in https://github.com/ciderapp/Cider/pull/154
* UI Changes and Feature Additions by @JaxonTekk in https://github.com/ciderapp/Cider/pull/153
* fixed "Confirm?" icons and switched PNG's to SVG's by @gms10ur in https://github.com/ciderapp/Cider/pull/159
* volume change fix by @JaxonTekk in https://github.com/ciderapp/Cider/pull/160
* Why quack by @yazninja in https://github.com/ciderapp/Cider/pull/167
* Added quit button for the webremote by @gms10ur in https://github.com/ciderapp/Cider/pull/168
* Center Web Remote Landscape by @cyraki in https://github.com/ciderapp/Cider/pull/170
* Create LICENSE by @cryptofyre in https://github.com/ciderapp/Cider/pull/171
* ui changes by @JaxonTekk in https://github.com/ciderapp/Cider/pull/173
* Added TR Language file & Fixed infinity symbol centering by @gms10ur in https://github.com/ciderapp/Cider/pull/175
* Fix playlist queue problem by @N0chteil in https://github.com/ciderapp/Cider/pull/178
* Upcoming Localization Commit by @maikirakiwi in https://github.com/ciderapp/Cider/pull/180
* Add Greek Translation by @down-bad in https://github.com/ciderapp/Cider/pull/179
* The OCD Update by @gms10ur in https://github.com/ciderapp/Cider/pull/181
* Massive i18n push by @maikirakiwi in https://github.com/ciderapp/Cider/pull/182
* Revert "Massive i18n push" by @cryptofyre in https://github.com/ciderapp/Cider/pull/183
* Revert "Massive i18n push" by @vapormusic in https://github.com/ciderapp/Cider/pull/184
* i18n update by @maikirakiwi in https://github.com/ciderapp/Cider/pull/185
* Some little fixes for WebRemote by @gms10ur in https://github.com/ciderapp/Cider/pull/186
* [i18n] ja_JP minor corrections by @maikirakiwi in https://github.com/ciderapp/Cider/pull/191
* Update Chinese translation by @EricZhang456 in https://github.com/ciderapp/Cider/pull/190
* ja_JP minor update by @noyciy7037 in https://github.com/ciderapp/Cider/pull/193
* Create zh_HK.jsonc by @kyw504100 in https://github.com/ciderapp/Cider/pull/194
* Update Greek Translation by @down-bad in https://github.com/ciderapp/Cider/pull/197
* Update on zh_HK translation by @kyw504100 in https://github.com/ciderapp/Cider/pull/199
* Added hu_HU (Hungarian) translation file by @Greenoliv in https://github.com/ciderapp/Cider/pull/201
* Update on zh_HK by @kyw504100 in https://github.com/ciderapp/Cider/pull/200
* Update tr_TR.jsonc by @gms10ur in https://github.com/ciderapp/Cider/pull/202
* A bunch of i18n changes by @maikirakiwi in https://github.com/ciderapp/Cider/pull/207
* Fix Overlapping items in playback-info/ Resize Queue in fs by @yazninja in https://github.com/ciderapp/Cider/pull/196
* Fix index.js by @yazninja in https://github.com/ciderapp/Cider/pull/209
* 99.999% i18n coverage for Cider. by @maikirakiwi in https://github.com/ciderapp/Cider/pull/208
* settings toggle for miniplayer to appear on top of windows. by @calumcl in https://github.com/ciderapp/Cider/pull/205
* Found the 0.001% of the untranslated strings lmao by @gms10ur in https://github.com/ciderapp/Cider/pull/211
* Update on zh_ translations by @kyw504100 in https://github.com/ciderapp/Cider/pull/210
* Update Greek Translation by @down-bad in https://github.com/ciderapp/Cider/pull/213
* HODOR HODOR HODOR  by @gms10ur in https://github.com/ciderapp/Cider/pull/216
* Pull Request of the day - 25 January 2022 by @maikirakiwi in https://github.com/ciderapp/Cider/pull/215
* Language option by @kyw504100 in https://github.com/ciderapp/Cider/pull/217
* i18n desc by @maikirakiwi in https://github.com/ciderapp/Cider/pull/218
* Another i18n desc by @kyw504100 in https://github.com/ciderapp/Cider/pull/219
* Language: en_SGA + fixed a typo by @kyw504100 in https://github.com/ciderapp/Cider/pull/220
* daily i18n files update by @gms10ur in https://github.com/ciderapp/Cider/pull/221
* Added a string + Updated zh_hk lang file by @kyw504100 in https://github.com/ciderapp/Cider/pull/222
* i18n sync by @maikirakiwi in https://github.com/ciderapp/Cider/pull/223
* Update Greek update update update by @down-bad in https://github.com/ciderapp/Cider/pull/225
* Add Marquee Back by @yazninja in https://github.com/ciderapp/Cider/pull/228
* Pull Request of the day - 26 January 2022 by @maikirakiwi in https://github.com/ciderapp/Cider/pull/229
* Formatting + i18n by @maikirakiwi in https://github.com/ciderapp/Cider/pull/230
* Daily PR - 27.01 (READY TO MERGE) by @gms10ur in https://github.com/ciderapp/Cider/pull/231
* [WIP] DiscordRPC - Add Open in Apple Music Button by @yazninja in https://github.com/ciderapp/Cider/pull/233
* Fix discordRPC by @yazninja in https://github.com/ciderapp/Cider/pull/234
* [DiscordRPC] Add storefront/ Fix songs with no catalog ID by @yazninja in https://github.com/ciderapp/Cider/pull/236
* Equalizer Presets by @maikirakiwi in https://github.com/ciderapp/Cider/pull/235
* Equalizer Preset + i18n by @maikirakiwi in https://github.com/ciderapp/Cider/pull/242
* Irregular update 29/01 by @kyw504100 in https://github.com/ciderapp/Cider/pull/241
* its like three i18n strings just merge by @maikirakiwi in https://github.com/ciderapp/Cider/pull/246
* Update Hungarian Language files by @Greenoliv in https://github.com/ciderapp/Cider/pull/248
* Someone forgot about storefront lol by @yazninja in https://github.com/ciderapp/Cider/pull/250
* Adding Cantonese translation by @tszngaiyip in https://github.com/ciderapp/Cider/pull/251
* Update Cantonese Translation by @StrikeSNC in https://github.com/ciderapp/Cider/pull/252
* Daily PR - 29.01.2022 by @gms10ur in https://github.com/ciderapp/Cider/pull/249
* Fix missing comma by @StrikeSNC in https://github.com/ciderapp/Cider/pull/253
* Develop by @coredev-uk in https://github.com/ciderapp/Cider/pull/254
* Update Greek Translation by @down-bad in https://github.com/ciderapp/Cider/pull/255
* Add French language by @ErwanGit in https://github.com/ciderapp/Cider/pull/258
* Add german translation by @motz0815 in https://github.com/ciderapp/Cider/pull/256
* Added create new playlist option to Add To Playlist context menu by @Swiftzerr in https://github.com/ciderapp/Cider/pull/262
* I fucked up last time i tried to do localization and didnt realize that I made a error by @Swiftzerr in https://github.com/ciderapp/Cider/pull/263
* Unbloat user menu by @Swiftzerr in https://github.com/ciderapp/Cider/pull/264
* Irregular update 30/01 by @kyw504100 in https://github.com/ciderapp/Cider/pull/265
* Daily PR 30.01.2022 - Ready to Merge by @gms10ur in https://github.com/ciderapp/Cider/pull/267
* CircleCI Support. by @cryptofyre in https://github.com/ciderapp/Cider/pull/269
* Text Ellipsis for song title by @yazninja in https://github.com/ciderapp/Cider/pull/268
* deez nutz by @maikirakiwi in https://github.com/ciderapp/Cider/pull/270
* Updated and fixed Hungarian translation by @BenjaminStonawski in https://github.com/ciderapp/Cider/pull/276
* en_PISS by @SoNothingMC in https://github.com/ciderapp/Cider/pull/275
* Changed some translations by @BenjaminStonawski in https://github.com/ciderapp/Cider/pull/277
* Italian translation by @ConnectedR in https://github.com/ciderapp/Cider/pull/279
* Supplemented with the new terms in 1.1.0 by @BenjaminStonawski in https://github.com/ciderapp/Cider/pull/278
* Resolved issue #281 by @TreeHouseFalcon in https://github.com/ciderapp/Cider/pull/286
* Translate to Spanish by @fl220 in https://github.com/ciderapp/Cider/pull/283
* Rename es_ES to es_ES.jsonc by @fl220 in https://github.com/ciderapp/Cider/pull/287
* Irregular update 01/02 by @kyw504100 in https://github.com/ciderapp/Cider/pull/280
* Update es_ES by @fl220 in https://github.com/ciderapp/Cider/pull/288
* EQ backend for qwack by @Swiftzerr in https://github.com/ciderapp/Cider/pull/289
* QOL of things by @maikirakiwi in https://github.com/ciderapp/Cider/pull/290
* Updated translations with new terms by @BenjaminStonawski in https://github.com/ciderapp/Cider/pull/292
* New translations for es_ES by @fl220 in https://github.com/ciderapp/Cider/pull/291
* Develop by @coredev-uk in https://github.com/ciderapp/Cider/pull/297
* i18n: Changed English (UK) into a main language rather than a fun language by @inalone in https://github.com/ciderapp/Cider/pull/299
* Update sv_SE by @ThisIsMemeboi in https://github.com/ciderapp/Cider/pull/301
* Updated some translations by @BenjaminStonawski in https://github.com/ciderapp/Cider/pull/306
* Added Romanian Translation by @georgeccotea in https://github.com/ciderapp/Cider/pull/307
* Update README to show winget by @yazninja in https://github.com/ciderapp/Cider/pull/314

## New Contributors
* @yazninja made their first contribution in https://github.com/ciderapp/Cider/pull/110
* @saboooor made their first contribution in https://github.com/ciderapp/Cider/pull/120
* @Swiftzerr made their first contribution in https://github.com/ciderapp/Cider/pull/119
* @gms10ur made their first contribution in https://github.com/ciderapp/Cider/pull/135
* @SoNothingMC made their first contribution in https://github.com/ciderapp/Cider/pull/143
* @JaxonTekk made their first contribution in https://github.com/ciderapp/Cider/pull/144
* @cyraki made their first contribution in https://github.com/ciderapp/Cider/pull/149
* @cryptofyre made their first contribution in https://github.com/ciderapp/Cider/pull/151
* @maikirakiwi made their first contribution in https://github.com/ciderapp/Cider/pull/154
* @down-bad made their first contribution in https://github.com/ciderapp/Cider/pull/179
* @EricZhang456 made their first contribution in https://github.com/ciderapp/Cider/pull/190
* @noyciy7037 made their first contribution in https://github.com/ciderapp/Cider/pull/193
* @Greenoliv made their first contribution in https://github.com/ciderapp/Cider/pull/201
* @calumcl made their first contribution in https://github.com/ciderapp/Cider/pull/205
* @tszngaiyip made their first contribution in https://github.com/ciderapp/Cider/pull/251
* @StrikeSNC made their first contribution in https://github.com/ciderapp/Cider/pull/252
* @coredev-uk made their first contribution in https://github.com/ciderapp/Cider/pull/254
* @ErwanGit made their first contribution in https://github.com/ciderapp/Cider/pull/258
* @motz0815 made their first contribution in https://github.com/ciderapp/Cider/pull/256
* @BenjaminStonawski made their first contribution in https://github.com/ciderapp/Cider/pull/276
* @ConnectedR made their first contribution in https://github.com/ciderapp/Cider/pull/279
* @TreeHouseFalcon made their first contribution in https://github.com/ciderapp/Cider/pull/286
* @fl220 made their first contribution in https://github.com/ciderapp/Cider/pull/283
* @inalone made their first contribution in https://github.com/ciderapp/Cider/pull/299
* @ThisIsMemeboi made their first contribution in https://github.com/ciderapp/Cider/pull/301
* @georgeccotea made their first contribution in https://github.com/ciderapp/Cider/pull/307

**Comment by elliotjarnit** [comment]
Fire changelog

**Comment by kyw504100** [comment]
How come an auto-gened log can missed my name. 🤣
I'm a fringe 🤣
